src/App.js

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./componenten/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [selection, setSelection] = useState([]);
+    const [endpoint, setEndpoint] = useState('https://pokeapi.co/api/v2/pokemon');
+
+    useEffect(() => {
+        async function getSelection() {
+            try {
+                const result = await axios.get(`${endpoint}`);
+                setSelection(result.data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        getSelection();
+    }, [endpoint]);
+
+    return (
+        <>
+            <div className="pokemon-selection">
+                {selection.results && selection.results.map((pokemon) => {
```

Comment:
Hier kan het dan iets simpeler
```suggestion
                {selection && selection.results.map((pokemon) => {
```